sequencer/src/context.rs

```diff
@@ -299,3 +294,47 @@ async fn handle_events<Ver: StaticVersionType>(
         }
     }
 }
+
+#[derive(Debug, Default)]
+pub struct TaskList(Vec<(String, JoinHandle<()>)>);
```

Comment:
I recently created #1341 because I think we expose a lot more than we would want to. If we all agree on that we should probably at the very least not unnecessarily expose new types.

I think we could also say we don't make any guarantees in terms of public API stability because at the moment this is first and foremost an application crate. In that case we should probably put this in the readme though.

Reply:
I can make this one `pub(crate)`, and add a disclaimer to the README